lib/tasks/rekey_encrypted_fields.rake

```diff
@@ -0,0 +1,77 @@
+desc 'Rekey (change keys) of all encrypted fields'
+task rekey: :environment do
+  old_key = Rails.application.secrets[:attr_encrypted_key_old]
+  new_key = Rails.application.secrets[:attr_encrypted_key]
+
+  hash_model_to_columns = Util::AttrEncrypted.get_all_encrypted_fields
+
+  ActiveRecord::Base.connection.truncate(UserAuthenticationToken.table_name)
+
+  hash_model_to_columns.except(UserAuthenticationToken).each do |model, encrypted_fields|
+    encrypted_fields.each do |field|
+      puts "Rekeying #{model} #{field}"
+      model.find_in_batches(batch_size: 10000) do |batch_of_records|
```

Comment:
Would you be able to sort this so we'll know where to restart from if this function fails mid-way rather than starting from a seemingly random point?

Reply:
To add on, it's preferable that we store the value into Redis so we know the last executed object and ID

Reply:
I can add an ordering, sure, but the intention is just to rerun it as it should be quick enough.

Reply:
Done.